src/vhdl/mega65r4.xdc

```diff
@@ -122,7 +122,7 @@ set_property -dict {PACKAGE_PIN C13 IOSTANDARD LVCMOS33} [get_ports kb_io2]
 set_property -dict {PACKAGE_PIN B13 IOSTANDARD LVCMOS33} [get_ports kb_jtagen]
 # Place Keyboard close to I/O pins
 create_pblock pblock_kbd0
-add_cells_to_pblock pblock_kbd0 [get_cells [list kbd0]]
+add_cells_to_pblock pblock_kbd0 [get_cells [list kbd0] [list mk2]]
```

Comment:
Hmmm? Why not just do [list kbd0 mk2]?

Reply:
I must confess I didn't check the syntax.
I just ran vivado_check and the build completed.